tasks/vm.py

```diff
@@ -32,14 +33,18 @@ def create_vm(
     debug: Optional[bool] = False,
     os_family: Optional[str] = None,
     use_fakeintake: Optional[bool] = True,
+    ami_id: Optional[str] = None,
 ) -> None:
     """
     Create a new virtual machine on the cloud.
     """
 
     extra_flags = {}
-    os_family = _get_os_family(os_family)
+    os_family, os_arch = _get_os_information(os_family, ami_id)
```

Comment:
Note: What do you think about calling `_get_os_information` only when `ami_id is not None`? It allows the function to have a better signature by changing the return type to `Tuple[str, Optional[str]]` and not calling `_get_os_family`.

Reply:
Sorry, I  meant `Tuple[str, str]` instead of `Tuple[str, Optional[str]]`